uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/SicknessRequestTest.php

```diff
@@ -38,29 +38,29 @@ public function testCreateAlsoCreateTheLeaveRequestBalanceChanges() {
 
     $sicknessReasons = array_flip(SicknessRequest::buildOptions('reason', 'validate'));
 
-    // a 7 days request, from friday to thursday
+    // a 9 days request, from thursday to friday next week
     $sicknessRequest = $service->create([
       'type_id' => 1,
       'contact_id' => $contact['id'],
       'status_id' => 1,
-      'from_date' => CRM_Utils_Date::processDate('2016-01-01'),
+      'from_date' => CRM_Utils_Date::processDate('2016-02-11'),
       'from_date_type' => $this->getLeaveRequestDayTypes()['All Day']['value'],
-      'to_date' => CRM_Utils_Date::processDate('2016-01-07'),
+      'to_date' => CRM_Utils_Date::processDate('2016-02-19'),
       'to_date_type' => $this->getLeaveRequestDayTypes()['All Day']['value'],
       'reason' => $sicknessReasons['accident']
     ], false);
 
     $leaveRequest = LeaveRequest::findById($sicknessRequest->leave_request_id);
 
     $balance = LeaveBalanceChange::getTotalBalanceChangeForLeaveRequest($leaveRequest);
-    // Since the 40 hours work pattern was used, and it this is a week long
-    // leave request, the balance will be 5 (for the 5 working days)
-    $this->assertEquals(5, $balance);
+    // Since the 40 hours work pattern was used and there are 7 working days during
+    // the 9 days period of the requests, the balance will be -7
+    $this->assertEquals(-7, $balance);
 
     $balanceChanges = LeaveBalanceChange::getBreakdownForLeaveRequest($leaveRequest);
-    // Even though the balance is 5, we must have 7 balance changes, one for
+    // Even though the balance is -5, we must have 7 balance changes, one for
```

Comment:
This should be "Even though the balance is -7, we must have 9 balance changes, one for"